### ISSUE TITLE
Office 365 application is not shown when a policy includes other applications in addition to office 365

### ISSUE THREAD

**Issue by aspnet4you** [merill/idPowerToys#6]
Nice tool but it requires a paid license!

I was able to do my first run and noticed that Office 365 application is not shown when a policy includes other applications in addition to office 365. Office 365 shows okay when it is the only app in the CA policy.

Is it a bug or some settings that I missed?

**Comment by merill** [comment]
Are you referring to Syncfusion? They do have a great community license that you can get depending on the size of the company. 

For O365, yes I only show the icon if it's the only app selected. If but it is listed below under selected apps along with the rest of the apps.

**Comment by aspnet4you** [comment]
Yes, I was referring to syncfusion license. I will try community license in my personal capacity.

O365 icon does not show up under app section when O365 app is combined with other apps. Didn't work for me when I ran with the latest code.

**Comment by abitlegacy** [comment]
+1 for this - also was affected by it. O365 by itself works just fine, but O365 + other applications only shows the other applications and not O365. This was with the Desktop Client.
